Readme.md

```diff
@@ -287,7 +287,7 @@ TIPS
    `export PARALLEL_TEST_FIRST_IS_1=true` will provide the same result
  - [email_spec and/or action_mailer_cache_delivery](https://github.com/grosser/parallel_tests/wiki)
  - [zeus-parallel_tests](https://github.com/sevos/zeus-parallel_tests)
- - [Distributed parallel test (e.g. Travis Support)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-and-Travis-Support)
+ - [Distributed Parallel Tests on CI systems (Travis, GitLab)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-on-CI-systems-(Travis,-GitLab))
```

Comment:
can we name it "CI Systems" ? ... otherwise it's just going to get longer forever :D
```suggestion
 - [Distributed Parallel Tests on CI systems (Travis, GitLab, etc)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-on-CI-systems)
```

Reply:
I agree, I've instead added Travis and GitLab in the text on the side (so it's still easy to find) and added another note since the distributed environment potentially allows to be integrated that way without supporting multiple-databases on one host.